convert2rhel/pkghandler.py

```diff
@@ -594,13 +605,13 @@ def remove_pkgs_unless_from_redhat(pkgs_to_remove, backup=True):
     """
     if not pkgs_to_remove:
         loggerinst.info("\nNothing to do.")
-        return
+        return []
```

Comment:
Codecov claims not covered by tests. It is a branch we should make sure we enter